### ISSUE TITLE
Getting an error with a lookup of with this plugin amazon.aws.aws_secret on an 6.0.0

### ISSUE THREAD

**Issue by HardHero** [ansible-collections/amazon.aws#1559]
### Summary

Using the latest of release of 6.0.0, I am trying to to a lookup of a secret in aws secrets manager like this:

`- name: "Set Rapid7 Token"
    set_fact:
       token: "{{ lookup('amazon.aws.aws_secret', 'path/to/token', region=PrimaryRegion, bypath=true) }}"`

This is throwing an error like this:

<img width="1189" alt="image" src="https://github.com/ansible-collections/amazon.aws/assets/16921446/e03c26e0-cb21-42cd-83f8-731508b11c07">

I have filled in below to the best of my knowledge, I also sanitized it for any revealing info, so bear with me there. this seems like a bug

### Issue Type

Bug Report

### Component Name

amazon.aws.aws_secret lookup plugin

### Ansible Version

```
ansible [core 2.11.12]
  config file = /tmp/provision/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /tmp/provision/.virtualenv/lib/python3.7/site-packages/ansible
  ansible collection location = /tmp/provision/.vendor
  executable location = /tmp/provision/.virtualenv/bin/ansible
  python version = 3.7.16 (default, Mar 10 2023, 03:25:26) [GCC 7.3.1 20180712 (Red Hat 7.3.1-15)]
  jinja version = 3.1.2
  libyaml = True

```

### Collection Versions

```console (paste below)
$ ansible-galaxy collection list
Collection         Version
------------------ -------
amazon.aws         6.0.0
ansible.posix      1.5.4
community.aws      6.0.0
community.general  7.0.0
team.ops 1.0.0
```

### AWS SDK versions

```console (paste below)
$ pip show boto boto3 botocore
WARNING: Package(s) not found: boto
Name: boto3
Version: 1.26.136
Summary: The AWS SDK for Python
Home-page: https://github.com/boto/boto3
Author: Amazon Web Services
Author-email:
License: Apache License 2.0
Location: /tmp/provision/.virtualenv/lib/python3.7/site-packages
Requires: botocore, jmespath, s3transfer
Required-by: voyant
---
Name: botocore
Version: 1.29.136
Summary: Low-level, data-driven core of boto 3.
Home-page: https://github.com/boto/botocore
Author: Amazon Web Services
Author-email:
License: Apache License 2.0
Location: /tmp/provision/.virtualenv/lib/python3.7/site-packages
Requires: jmespath, python-dateutil, urllib3
Required-by: boto3, s3transfer
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed
CALLBACKS_ENABLED(/tmp/provision/ansible.cfg) = ['ansible.posix.profile_tasks']
COLLECTIONS_PATHS(/tmp/provision/ansible.cfg) = ['/tmp/provision/.vendor']
DEFAULT_ROLES_PATH(/tmp/provision/ansible.cfg) = ['/tmp/provision/.vendor']
INTERPRETER_PYTHON(/tmp/provision/ansible.cfg) = {{ ansible_playbook_python }}
LOCALHOST_WARNING(/tmp/provision/ansible.cfg) = False
```

### OS / Environment

Amazon Linux 2

### Steps to Reproduce

```yaml (paste below)
- name: set region
  set_fact:
    PrimaryRegion: 'us-west-2'

- name: "Set Rapid7 Token"
  set_fact:
    token: "{{ lookup('amazon.aws.aws_secret', 'path/to/token', region=PrimaryRegion, bypath=true) }}"
```

### Expected Results

I expected the see the secret set to that `token` and the playbook to continue.

If I set this collection back to `5.4.0` this task completes as expected.

### Actual Results

```console (paste below)
TASK [team.ops.bootstrap : Set Token] *************************
Thursday 18 May 2023  22:21:28 +0000 (0:00:00.524)       0:01:52.284 **********
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'amazon.aws.aws_secret'. Error was a <class 'AttributeError'>, original message: 'LookupModule' object has no attribute 'get_options'"}
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by tremble** [comment]
Thanks for taking the time to open this Issue. With release 6.0.0 we've dropped support for Ansible 2.11.

Please see:
https://github.com/ansible-collections/amazon.aws/blob/stable-6/README.md#ansible-version-compatibility
and:
https://github.com/ansible-collections/amazon.aws/blob/stable-6/meta/runtime.yml

I would also add that Ansible Core 2.11 went end of life in November, and no longer receives any support or security fixes.

**Comment by tremble** [comment]
Closing, won't fix. Ansible 2.11 not supported by 6.0.0.

**Comment by HardHero** [comment]
Thank so much :)
